com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/HTTPBulkRequest.spl

```diff
@@ -0,0 +1,125 @@
+namespace com.ibm.streamsx.inet.http;
+
+/**
+ * The operator HTTPBulkRequest implements http bulk operations and processes the results asynchronous from the input stream.
+ * The operator opens on request up to *maxConnections*parallel http connections an sends the results downstream once the 
+ * response is received.
+
+ * @input	Request				The request Stream with attributes of the http request
+ * @output	Result				The result stream. One tuple is generated for each input tuple.
+ * 								The minimum type is : tuple<rstring body, list<rstring> headers, rstring message, int32 status, rstring url> where
+ * 								body is the response body, headers is the response header list, message is the error/success message, status is the error status of the operation, url is the encoded url
+ * @output	Error				The error stream. One tuple indicates a failed http operation.
+ * 								The type is : <rstring url, rstring message, int32 status>.
+ * 
+ * @param	method				This required parameter determines the kind of the http request. Type HttpMethod
+ * @param	url					The request url. Required parameter.
+ * @param	urlEncode			Defines the function to be used to prepare the url. Useful values are *passThrough* or [function:com.ibm.streamsx.inet.http::urlEncode(rstring)] . Default is urlEncode.
+ * @param	data				The body of the operation. Default is the empty string.
+ * @param	headers				http headers passes in the http request. Default is the empty list.
+ * @param	contentTypeHeader	The content type header. If this header is not empty, it is appended to the headers list. Default is "ContentType: application/x-www-form-urlencoded"
+ * @param	requestTimeout		This is the maximum time in seconds that you allow the transfer operation to take. Normally, name lookups can take a considerable time and limiting operations to less than a few minutes risk aborting perfectly normal operations. Default timeout is 5
+ * @param	connectionTimeout	This is the maximum time in seconds that you allow the connection to the server to take. This only limits the connection phase, once it has connected, this option is of no more use. Default is 20 seconds
+ * @param	width				The number of parallel used threads. This conforms to the maximum number if used http connections. Default is 8
+ * @param	queue				The length of the input queue ot each thread. Default 1
+ * @param	congestionPolicy	The congestion policy of the input queue. Type *Sys.CongestionPolicy*. Default Sys.Wait
+ * @param	user				If this string is not empty, this user name is passed to to the server.
```

Comment:
Probably better to reorder the description so that the purpose comes first, rather than "If this string is not empty, ...", e.g.

_User name for authentication. Ignored if empty._

Reply:
yes I will change